index.html

```diff
@@ -2,23 +2,74 @@
 <html lang="en">
     <meta charset="UTF-8">   
     <head>
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no">
         <title> Save the planet</title>
+        <script src="menu.js"></script>
         <link href="./CSS/homestyle.css" rel=stylesheet>         
     </head>
-    <body>
+    <body>       
         <section class="hero">
-            <div class="hero__image">
-                <div class="hero__shape"><img  src="./Photos/Shape.png" alt="Photo of the logo"></div>
+            <div class="hero__navbar">
+                <ul class="hero__navbar--list">
+                    <li ><a href="#" class="hero__navbar--item">Home</a></li>
+                    <li class="hero__navbar--dropdown">
```

Comment:
aici la fel ca mai sus, poti folosi hero__dropdown, si in interior incepi cu clasele de dropdown__numeElement